rust/settings/BUILD.bazel

```
@@ -253,3 +253,11 @@ per_crate_rustc_flag(
     build_setting_default = [],
     visibility = ["//visibility:public"],
 )
+
+# A flag to control whether to include data files in compile_data.
+incompatible_flag(
+    name = "incompatible_do_not_include_data_in_compile_data",
+    build_setting_default = False,
```

Let's default this to true - people can set it to False if they need time to migrate.

```suggestion
    build_setting_default = True,
```